src/initialization/Warnings.cpp

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2022 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Marco Garten
+ * License: BSD-3-Clause-LBNL
+ */
+#include "ImpactX.H"
+
+#include <ablastr/warn_manager/WarnManager.H>
+
+#include <AMReX_BLProfiler.H>
+#include <AMReX_ParmParse.H>
+
+#include <optional>
+#include <stdexcept>
+#include <string>
+
+
+void impactx::ImpactX::init_warning_logger ()
```

Comment:
Minimally more compact:
```suggestion
namespace impactx
{
void ImpactX::init_warning_logger ()
```